libraries/resource_hab_sup_windows.rb

```diff
@@ -51,15 +51,15 @@ class HabSupWindows < HabSup
 
         hab_package 'core/windows-service' do
           bldr_url new_resource.bldr_url if new_resource.bldr_url
-          version hab_windows_service_version
+          version new_resource.service_version if new_resource.service_version
         end
 
         execute 'hab pkg exec core/windows-service install' do
           not_if { ::Win32::Service.exists?('Habitat') }
         end
 
-        template 'C:/hab/svc/windows-service/HabService.exe.config' do
-          source 'windows/HabService.exe.config.erb'
+        template 'C:/hab/svc/windows-service/HabService.dll.config' do
```

Comment:
So should this be a property in case it changes in future for easy change?

Otherwise this should be a var as you are referencing the same file on line 75

Reply:
Good point, I can do that pretty easily

Reply:
Well, not sure how I could best go about this. According to the habitat team, this is how it's always been configured I believe. it was wrong previously which I was unaware of and subsequent updates were done incorrectly due to lack of testing as well as misunderstanding on my end. @mwrock can maybe chime in here. He may be able to shed light on wether it's a worry or not.

Reply:
will need to discuss this one more internally and circle back

Reply:
corrections made according to @Xorima and I collaborating on the proper syntax. Will merge upon test completion.